### PR TITLE
Add wtETH OP V3 - draft

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: content/aips/ADD-OPV3WSTETH.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: onboard WSTETH to Aave Optimism V3 pool
```
Comment (sakulstra) [inline content/aips/ADD-OPV3WSTETH.md:2]:
```suggestion
title: Add WSTETH to Aave Optimism V3 pool
```
perhaps better be consistent, doesn't matter though

File: content/aips/ADD-OPV3WSTETH.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+|Liquidation Protocol Fee|10%|
+|Borrow Cap|940|
+|Supply Cap|6000|
+|Debt Ceiling|N/A|
+|Base|0.25%|
+|Slope1|4.5%|
+|Uoptimal|45%|
+|Slope2|80%|
+
+The proposal payload uses the [GenericListingEngine](https://etherscan.io/address/0xC51e6E38d406F98049622Ca54a6096a23826B426#code) to perform a new asset listing with the following parameters:
```
Comment (sakulstra) [inline content/aips/ADD-OPV3WSTETH.md:46]:
```suggestion
The proposal payload uses the [GenericListingEngine](https://optimistic.etherscan.io/address/0x7b8Fa4540246554e77FCFf140f9114de00F8bB8D#code) to perform a new asset listing with the following parameters:
```

File: content/aips/ADD-OPV3WSTETH.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      rateStrategy: address(0), // TODO: deploy rate strategy
+      enabledToBorrow: true,
+      stableRateModeEnabled: false,
+      borrowableInIsolation: false,
+      withSiloedBorrowing: false,
+      flashloanable: true,
+      ltv: 70_00,
+      liqThreshold: 79_00,
+      liqBonus: 7_20,
+      reserveFactor: 10_00,
```
Comment (sakulstra) [inline content/aips/ADD-OPV3WSTETH.md:62]:
```suggestion
      reserveFactor: 15_00,
```
should be 15 according to aip

File: content/aips/MAIPolyV3CapUpdate.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (sakulstra) [inline content/aips/MAIPolyV3CapUpdate.md:1]: plz remove this file

**Comment by marczeller** [comment]
ok done thx

**Pull request closed without merging**